.dojorc

```diff
@@ -6,7 +6,7 @@
 	"build-widget": {
 		"prefix": "dojo",
 		"widgets": [
-			"src/accordion-pane",
+			"src/accordion",
```

Comment:
`AccordionPane` was renamed `Accordion` throughout, as this felt more appropriate now that a `Pane` is also exported.